python/extractor/tsg-python/tsp/Cargo.toml

```diff
@@ -24,6 +24,7 @@ include = [
 [lib]
 path = "bindings/rust/lib.rs"
 
+## When updating these dependencies, run `misc/bazel/3rdparty/update_cargo_deps.sh`
```

Comment:
This one gets a double comment! I assume there's no semantic significance. 🙂

Reply:
Does it? I tried to add the comment to every `Cargo.toml` file that contributes to an extractor build, with the exception of ql-for-ql, which we don't build with bazel.

Reply:
I meant compared to the other `Cargo.toml` file, where the added line starts with a single comment character. Sorry for the confusing wording.

Reply:
Ah, double shebang! Yeah that wasn't intentional, but shouldn't change anything either.